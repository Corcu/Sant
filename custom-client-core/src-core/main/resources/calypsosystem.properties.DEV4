#Fri Mar 11 09:35:25 CET 2022
CALYPSO_MESSAGING_SERVER_URL=failover\:(tcp\://localhost\:62616)?timeout\=30000
CALYPSO_TRANSIENT_MESSAGING_SERVER_URL=failover\:(tcp\://localhost\:63616)?timeout\=30000
CLIENTPASSWD=FBlocaldev123
CLIENTUSER=calypso_local_dev
CUSTOM_SSL_TRUSTSTORE_FILE="calypso_client.truststore"
CUSTOM_SSL_TRUSTSTORE_PWD="caltrust"
DATASERVER_NAMING_PROVIDER_URL=http-remoting\://BVMNGLCALYPD111.nngg.corp\:14000
DATA_SERVER_NAME=DEV4_DS
DBPASSWORD=Cal_cog0
DBURL=jdbc\:oracle\:thin\:@(DESCRIPTION\=(ADDRESS\=(PROTOCOL\=TCP)(HOST\=dcalco4d.nngg.corp)(PORT\=1530))(CONNECT_DATA\=(SERVICE_NAME\=dcalco4d)))
DBUSER=calypso
DRIVER=oracle.jdbc.OracleDriver
DU_MESSAGING_SERVER_URL=tcp\://localhost\:61917
EDEALING_LLMEVENTSERVERURL=tcp\://localhost\:6199
ENGINESERVER_NAMING_PROVIDER_URL=http-remoting\://BVMNGLCALYPD111.nngg.corp\:14300,http-remoting\://BVMNGLCALYPD111.nngg.corp\:14400,http-remoting\://BVMNGLCALYPD111.nngg.corp\:14200
EVENTSERVER_NAMING_PROVIDER_URL=http-remoting\://BVMNGLCALYPD111.nngg.corp\:14100
JMS_NAMING_PROVIDER_URL=http-remoting\://BVMNGLCALYPD111.nngg.corp\:14100
LIQUIDITY_SERVER_BUS_URL=tcp\://localhost\:61620
LLM_CURVE_QUOTE_SERVER_URL=tcp\://localhost\:8199
LLM_MAS_SERVER_URL=tcp\://localhost\:8199
LLM_QUOTE_SERVER_URL=tcp\://localhost\:6199
LOCALE=Spanish
LOG_CATEGORY=ALL
LOG_LEVEL=DEBUG,INFO,WARN,ERROR,FATAL
LOG_MAXFILESIZE=300000000
MAX_POSTINGS_PER_USER=10000
MAX_TASKS_PER_USER=250000
MAX_TRADES_PER_USER=10000
MAX_TRANSFERS_PER_USER=10000
MONITOR_SQL=true
MiddleTierDRIVER=
NDPERSISTOREVENTSERVERURL=tcp\://localhost\:1974
RATE_ENGINE_LLMEVENTSERVERURL=tcp\://localhost\:8199
RODATASERVER_NAMING_PROVIDER_URL=http-remoting\://BVMNGLCALYPD111.nngg.corp\:14040
RODS_MAX_EVENTS=30000
SERVERHOST=BVMNGLCALYPD111.nngg.corp
SERVERS=dataserver,eventserver,exp_engineserver,gen_engineserver,imp_engineserver
USE_CALL_ACCOUNT=true
USE_MESSAGE_AMOUNT_FORMAT=true
USE_READ_ONLY_DS=true
USE_SSL=false
