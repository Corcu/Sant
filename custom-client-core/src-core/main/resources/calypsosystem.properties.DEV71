#Thu Mar 30 10:18:48 CEST 2023
LOG_LEVEL=ALL
LLM_QUOTE_SERVER_URL=tcp\://localhost\:6199
AUTO_COMPENSATION_MODEL=CSDBased
SERVERS=dataserver,eventserver,exp_engineserver,gen_engineserver,imp_engineserver
LOCALE=Spanish
USE_CALL_ACCOUNT=true
LIQUIDITY_SERVER_BUS_URL=tcp\://localhost\:61620
NDPERSISTOREVENTSERVERURL=tcp\://localhost\:1974
LOG_MAXFILESIZE=300000000
LLM_MAS_SERVER_URL=tcp\://localhost\:8199
PartialSettlementNoCheck=true
CACHE_CLIENT_EXTENDED=true
USE_NUMERIC_FOR_RELA_REF=false
SPLITWINDOW_ENHANCED=true
LOG_AGGREGATION=false
DATASERVER_NAMING_PROVIDER_URL=https-remoting\://BVMNGLCALYPD117.nngg.corp\:11001
USE_PRODUCT_GROUP=true
USE_MESSAGE_AMOUNT_FORMAT=true
WRITE_OFF=TRUE
CHECK_QUANTITY_FOR_SECURITY_MATCHING=false
ENGINESERVER_NAMING_PROVIDER_URL=https-remoting\://BVMNGLCALYPD117.nngg.corp\:11301,https-remoting\://BVMNGLCALYPD117.nngg.corp\:11401,https-remoting\://BVMNGLCALYPD117.nngg.corp\:11201,https-remoting\://BVMNGLCALYPD117.nngg.corp\:11501
ALLOW_EXECUTE_SQL=true
CALYPSO_TRANSIENT_MESSAGING_SERVER_URL=failover\:(tcp\://localhost\:63616)?timeout\=30000
DATA_SERVER_NAME=DEV71_DS
CUSTOM_SSL_TRUSTSTORE_SERVER_PWD=calypso
RODS_MAX_EVENTS=30000
com.calypso.cache.client.extended=true
LLM_CURVE_QUOTE_SERVER_URL=tcp\://localhost\:8199
RODATASERVER_NAMING_PROVIDER_URL=https-remoting\://BVMNGLCALYPD117.nngg.corp\:11041
DEFAULT_ROWPREFETCH_SIZE=100
MONITOR_SQL=false
USE_SSL=true
SR_2020=true
USE_READ_ONLY_DS=true
JMS_NAMING_PROVIDER_URL=https-remoting\://BVMNGLCALYPD117.nngg.corp\:11101
PARTIAL_CASH_FOR_SECURITY_MATCHING=true
CALYPSO_MESSAGING_SERVER_URL=failover\:(tcp\://localhost\:62616)?timeout\=30000
AUTO_FEED_EXTERNAL_REF=false
USE_KAFKA=@USE_KAFKA@
USE_SR_2022=True
LOG_CATEGORY=GestorSTPSwiftMessageEngine
SERVERHOST=BVMNGLCALYPD117.nngg.corp
SEC_WITHHOLDINGTAX=true
MAIL_CLIENT_TYPE=calypsox.tk.util.email.SantanderMailSenderAdapter
EVENTSERVER_NAMING_PROVIDER_URL=https-remoting\://BVMNGLCALYPD117.nngg.corp\:11101
SKIP_INVENTORY_EXCEPTION=true
EDEALING_LLMEVENTSERVERURL=tcp\://localhost\:6199
WRITE_OFF_FOR_SECURITY_MATCHING=true
DU_MESSAGING_SERVER_URL=tcp\://localhost\:61917
RATE_ENGINE_LLMEVENTSERVERURL=tcp\://localhost\:8199