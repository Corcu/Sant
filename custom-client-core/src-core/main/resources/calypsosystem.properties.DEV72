#Calypso Client Login Defaults Mon Apr 24 12:12:40 CEST 2023
#Mon Apr 24 12:12:40 CEST 2023
LOCALE=Spanish
LOG_LEVEL=INFO,WARN,ERROR,FATAL
LLM_QUOTE_SERVER_URL=tcp\://localhost\:6199
AUTO_COMPENSATION_MODEL=CSDBased
SERVERS=dataserver,eventserver,exp_engineserver,gen_engineserver,imp_engineserver
USE_CALL_ACCOUNT=true
LIQUIDITY_SERVER_BUS_URL=tcp\://localhost\:61620
NDPERSISTOREVENTSERVERURL=tcp\://localhost\:1974
LOG_MAXFILESIZE=300000000
LLM_MAS_SERVER_URL=tcp\://localhost\:8199
PartialSettlementNoCheck=true
CACHE_CLIENT_EXTENDED=true
USE_NUMERIC_FOR_RELA_REF=false
SPLITWINDOW_ENHANCED=true
DATASERVER_NAMING_PROVIDER_URL=https-remoting\://BVMNGLCALYPD117.nngg.corp\:12001
USE_PRODUCT_GROUP=@USE_PRODUCT_GROUP@
USE_MESSAGE_AMOUNT_FORMAT=true
ENGINESERVER_NAMING_PROVIDER_URL=https-remoting\://BVMNGLCALYPD117.nngg.corp\:12301,https-remoting\://BVMNGLCALYPD117.nngg.corp\:12401,https-remoting\://BVMNGLCALYPD117.nngg.corp\:12201,https-remoting\://BVMNGLCALYPD117.nngg.corp\:12501
ALLOW_EXECUTE_SQL=true
CALYPSO_TRANSIENT_MESSAGING_SERVER_URL=failover\:(tcp\://localhost\:63616)?timeout\=30000
DOCUMENT_SECURITY=false
DATA_SERVER_NAME=DEV72_DS
CUSTOM_SSL_TRUSTSTORE_SERVER_PWD=calypso
RODS_MAX_EVENTS=30000
com.calypso.cache.client.extended=true
LLM_CURVE_QUOTE_SERVER_URL=tcp\://localhost\:8199
RODATASERVER_NAMING_PROVIDER_URL=https-remoting\://BVMNGLCALYPD117.nngg.corp\:12041
DEFAULT_ROWPREFETCH_SIZE=100
MONITOR_SQL=false
USE_SSL=true
USE_READ_ONLY_DS=true
JMS_NAMING_PROVIDER_URL=https-remoting\://BVMNGLCALYPD117.nngg.corp\:12101
CALYPSO_MESSAGING_SERVER_URL=failover\:(tcp\://localhost\:62616)?timeout\=30000
AUTO_FEED_EXTERNAL_REF=false
USE_SR_2022=True
LOG_CATEGORY=GestorSTPSwiftMessageEngine
SERVERHOST=BVMNGLCALYPD117.nngg.corp
EVENTSERVER_NAMING_PROVIDER_URL=https-remoting\://BVMNGLCALYPD117.nngg.corp\:12101
SKIP_INVENTORY_EXCEPTION=true
EDEALING_LLMEVENTSERVERURL=tcp\://localhost\:6199
DU_MESSAGING_SERVER_URL=tcp\://localhost\:61917
FEEDSERVER_PORT=6666
RATE_ENGINE_LLMEVENTSERVERURL=tcp\://localhost\:8199

