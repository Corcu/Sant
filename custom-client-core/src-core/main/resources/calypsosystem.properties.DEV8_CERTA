#Updated on Mon Jun 27 14:30:30 CEST 2022
#Mon Jun 27 14:30:30 CEST 2022
AUTO_FEED_EXTERNAL_REF=false
CACHE_CLIENT_EXTENDED=true
CALYPSO_MESSAGING_SERVER_URL=failover\:(tcp\://localhost\:62616)?timeout\=30000
CALYPSO_TRANSIENT_MESSAGING_SERVER_URL=failover\:(tcp\://localhost\:63616)?timeout\=30000
CHECK_QUANTITY_FOR_SECURITY_MATCHING=false
CLIENTPASSWD=FBlocaldev123
CLIENTUSER=calypso_local_dev
CUSTOM_SSL_TRUSTSTORE_SERVER_PWD=dev
DATASERVER_NAMING_PROVIDER_URL=https-remoting\://BVMNGLCALYPD113.nngg.corp\:18001
DATA_SERVER_NAME=CERT_AGILE_DS
DBPASSWORD=Cal_cog0
DBURL=jdbc\:oracle\:thin\:@(DESCRIPTION\=(ADDRESS\=(PROTOCOL\=TCP)(HOST\=dcalcoca.nngg.corp)(PORT\=1530))(CONNECT_DATA\=(SERVICE_NAME\=dcalcoca)))
DBUSER=calypso
DRIVER=oracle.jdbc.OracleDriver
DU_MESSAGING_SERVER_URL=tcp\://localhost\:61917
EDEALING_LLMEVENTSERVERURL=tcp\://localhost\:6199
ENGINESERVER_NAMING_PROVIDER_URL=https-remoting\://localhost\:18201
EVENTSERVER_NAMING_PROVIDER_URL=https-remoting\://BVMNGLCALYPD113.nngg.corp\:18101
JMS_NAMING_PROVIDER_URL=https-remoting\://BVMNGLCALYPD113.nngg.corp\:18101
LIQUIDITY_SERVER_BUS_URL=tcp\://localhost\:61620
LLM_CURVE_QUOTE_SERVER_URL=tcp\://localhost\:8199
LLM_MAS_SERVER_URL=tcp\://localhost\:8199
LLM_QUOTE_SERVER_URL=tcp\://localhost\:6199
LOCALE=Spanish
LOG_CATEGORY=
LOG_LEVEL=INFO,WARN,ERROR,FATAL
LOG_MAXFILESIZE=300000000
MAIL_CLIENT_TYPE=calypsox.tk.util.email.SantanderMailSenderAdapter
MONITOR_SQL=true
NDPERSISTOREVENTSERVERURL=tcp\://localhost\:1974
PARTIAL_CASH_FOR_SECURITY_MATCHING=true
RATE_ENGINE_LLMEVENTSERVERURL=tcp\://localhost\:8199
RODATASERVER_NAMING_PROVIDER_URL=https-remoting\://BVMNGLCALYPD113.nngg.corp\:18041
RODS_MAX_EVENTS=30000
SEC_WITHHOLDINGTAX=true
SERVERHOST=BVMNGLCALYPD113.nngg.corp
SERVERS=dataserver,eventserver,exp_engineserver,gen_engineserver,imp_engineserver
SPLITWINDOW_ENHANCED=true
SR_2020=true
USE_CALL_ACCOUNT=true
USE_MESSAGE_AMOUNT_FORMAT=true
USE_SSL=true
WRITE_OFF=TRUE
WRITE_OFF_FOR_SECURITY_MATCHING=true
com.calypso.cache.client.extended=true
CALYPSOSERVICESINTERFACESMS_NAMING_PROVIDER_URL=https-remoting\://localhost\:19081
