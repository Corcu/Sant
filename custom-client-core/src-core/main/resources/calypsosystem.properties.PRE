#Wed Mar 30 10:08:35 CEST 2022
ALLOW_EXECUTE_SQL=true
AUTOLOGIN_APPLICATIONS=engines,monitor
AUTO_FEED_EXTERNAL_REF=false
CACHE_CLIENT_EXTENDED=true
CALYPSO_MESSAGING_SERVER_URL=failover\:(tcp\://localhost\:62616)?timeout\=30000
CALYPSO_TRANSIENT_MESSAGING_SERVER_URL=failover\:(tcp\://localhost\:63616)?timeout\=30000
CHECK_QUANTITY_FOR_SECURITY_MATCHING=false
CLIENTPASSWD=FBlocaldev123
CLIENTUSER=calypso_local_dev
CUSTOM_SSL_TRUSTSTORE_SERVER_PWD=dev
DATASERVER_NAMING_PROVIDER_URL=https-remoting\://BVMNGLCALI111.nngg.corp\:11001
DATA_SERVER_NAME=PRE_DS
DBPASSWORD=
DBURL=jdbc\:oracle\:thin\:@(DESCRIPTION\=(ADDRESS\=(PROTOCOL\=TCP)(HOST\=pcalco.nngg.corp)(PORT\=1528))(CONNECT_DATA\=(SERVICE_NAME\=pcalco)))
DBUSER=calypso
DEFAULT_ROWPREFETCH_SIZE=100
DRIVER=oracle.jdbc.OracleDriver
DU_MESSAGING_SERVER_URL=tcp\://localhost\:61917
EDEALING_LLMEVENTSERVERURL=tcp\://localhost\:6199
ENGINESERVER_NAMING_PROVIDER_URL=https-remoting\://BVMNGLCALI117.nngg.corp\:11301,https-remoting\://BVMNGLCALI119.nngg.corp\:11401,https-remoting\://BVMNGLCALI115.nngg.corp\:11201,https-remoting\://BVMNGLCALI121.nngg.corp\:11501
EVENTSERVER_NAMING_PROVIDER_URL=https-remoting\://BVMNGLCALI111.nngg.corp\:11101
JMS_NAMING_PROVIDER_URL=https-remoting\://BVMNGLCALI111.nngg.corp\:11101
LIQUIDITY_SERVER_BUS_URL=tcp\://localhost\:61620
LLM_CURVE_QUOTE_SERVER_URL=tcp\://localhost\:8199
LLM_MAS_SERVER_URL=tcp\://localhost\:8199
LLM_QUOTE_SERVER_URL=tcp\://localhost\:6199
LOCALE=Spanish
LOG_CATEGORY=com.calypso.acadia.task.PollingTask,com.calypso.acadia.http.HttpRequestService
LOG_LEVEL=INFO,WARN,ERROR,FATAL
LOG_MAXFILESIZE=300000000
MONITOR_SQL=true
NDPERSISTOREVENTSERVERURL=tcp\://localhost\:1974
PARTIAL_CASH_FOR_SECURITY_MATCHING=true
RATE_ENGINE_LLMEVENTSERVERURL=tcp\://localhost\:8199
RODATASERVER_NAMING_PROVIDER_URL=https-remoting\://BVMNGLCALI111.nngg.corp\:11041
RODS_MAX_EVENTS=30000
SERVERHOST=BVMNGLCALI111.nngg.corp
SERVERS=dataserver,eventserver,exp_engineserver,gen_engineserver,imp_engineserver
SPLITWINDOW_ENHANCED=true
USE_CALL_ACCOUNT=true
USE_MESSAGE_AMOUNT_FORMAT=true
USE_READ_ONLY_DS=true
USE_SSL=true
WRITE_OFF_FOR_SECURITY_MATCHING=true
com.calypso.cache.client.extended=true
