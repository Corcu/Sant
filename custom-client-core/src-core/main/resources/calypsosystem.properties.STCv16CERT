#Tue Mar 29 18:03:38 CEST 2022
AUTO_FEED_EXTERNAL_REF=false
CACHE_CLIENT_EXTENDED=true
CALYPSO_MESSAGING_SERVER_URL=failover\:(tcp\://localhost\:62616)?timeout\=30000
CALYPSO_TRANSIENT_MESSAGING_SERVER_URL=failover\:(tcp\://localhost\:63616)?timeout\=30000
CLIENTPASSWD=FBlocaldev123
CLIENTUSER=calypso_local_dev
DATASERVER_NAMING_PROVIDER_URL=http-remoting\://BVMNGLCALYPD111.nngg.corp\:11000
DATA_SERVER_NAME=CERT_DS
DBPASSWORD=Cal_cog0
DBURL=jdbc\:oracle\:thin\:@(DESCRIPTION\=(ADDRESS\=(PROTOCOL\=TCP)(HOST\=dcalco2d.nngg.corp)(PORT\=1528))(CONNECT_DATA\=(SERVICE_NAME\=dcalco2d)))
DBUSER=calypso
DEFAULT_ROWPREFETCH_SIZE=100
DRIVER=oracle.jdbc.OracleDriver
DU_MESSAGING_SERVER_URL=tcp\://localhost\:61917
EDEALING_LLMEVENTSERVERURL=tcp\://localhost\:6199
ENGINESERVER_NAMING_PROVIDER_URL=http-remoting\://BVMNGLCALYPD111.nngg.corp\:11300,http-remoting\://BVMNGLCALYPD111.nngg.corp\:11400,http-remoting\://BVMNGLCALYPD111.nngg.corp\:11200,http-remoting\://BVMNGLCALYPD111.nngg.corp\:11500
EVENTSERVER_NAMING_PROVIDER_URL=http-remoting\://BVMNGLCALYPD111.nngg.corp\:11100
JMS_NAMING_PROVIDER_URL=http-remoting\://BVMNGLCALYPD111.nngg.corp\:11100
LIQUIDITY_SERVER_BUS_URL=tcp\://localhost\:61620
LLM_CURVE_QUOTE_SERVER_URL=tcp\://localhost\:8199
LLM_MAS_SERVER_URL=tcp\://localhost\:8199
LLM_QUOTE_SERVER_URL=tcp\://localhost\:6199
LOCALE=Spanish
LOG_CATEGORY=com.calypso.acadia.task.PollingTask,com.calypso.acadia.http.HttpRequestService
LOG_LEVEL=INFO,WARN,ERROR,FATAL
LOG_MAXFILESIZE=300000000
MONITOR_SQL=true
NDPERSISTOREVENTSERVERURL=tcp\://localhost\:1974
RATE_ENGINE_LLMEVENTSERVERURL=tcp\://localhost\:8199
RODATASERVER_NAMING_PROVIDER_URL=http-remoting\://BVMNGLCALYPD111.nngg.corp\:11040
RODS_MAX_EVENTS=30000
SERVERHOST=BVMNGLCALYPD111.nngg.corp
SERVERS=dataserver,eventserver,exp_engineserver,gen_engineserver,imp_engineserver
SPLITWINDOW_ENHANCED=true
USE_CALL_ACCOUNT=true
USE_MESSAGE_AMOUNT_FORMAT=true
USE_READ_ONLY_DS=true
USE_SSL=false
com.calypso.cache.client.extended=true
