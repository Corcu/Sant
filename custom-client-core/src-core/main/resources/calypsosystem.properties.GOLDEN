#Thu Mar 30 09:28:44 CEST 2023
LOG_LEVEL=DEBUG,INFO,WARN,ERROR,FATAL
AUTOLOGIN_APPLICATIONS=engines,monitor
LLM_QUOTE_SERVER_URL=tcp\://localhost\:6199
AUTO_COMPENSATION_MODEL=CSDBased
SERVERS=eventserver,dataserver,rodataserver,exp_engineserver,gen_engineserver,imp_engineserver,cws_engineserver
LOCALE=Spanish
USE_CALL_ACCOUNT=true
MAX_POSTINGS_PER_USER=250000
LIQUIDITY_SERVER_BUS_URL=tcp\://localhost\:61620
NDPERSISTOREVENTSERVERURL=tcp\://localhost\:1974
LOG_MAXFILESIZE=300000000
LLM_MAS_SERVER_URL=tcp\://localhost\:8199
PartialSettlementNoCheck=true
CACHE_CLIENT_EXTENDED=true
SPLITWINDOW_ENHANCED=true
DATASERVER_NAMING_PROVIDER_URL=https-remoting\://BVMNGLCALYPD113.nngg.corp\:11001
USE_MESSAGE_AMOUNT_FORMAT=true
ENGINESERVER_NAMING_PROVIDER_URL=https-remoting\://BVMNGLCALYPD113.nngg.corp\:11301,https-remoting\://BVMNGLCALYPD113.nngg.corp\:11401,https-remoting\://BVMNGLCALYPD113.nngg.corp\:11201,https-remoting\://BVMNGLCALYPD113.nngg.corp\:11501
ALLOW_EXECUTE_SQL=true
CALYPSO_TRANSIENT_MESSAGING_SERVER_URL=failover\:(tcp\://localhost\:63616)?timeout\=30000
DATA_SERVER_NAME=MA_DS
CUSTOM_SSL_TRUSTSTORE_SERVER_PWD=calypso
RODS_MAX_EVENTS=30000
com.calypso.cache.client.extended=true
LLM_CURVE_QUOTE_SERVER_URL=tcp\://localhost\:8199
RODATASERVER_NAMING_PROVIDER_URL=https-remoting\://BVMNGLCALYPD113.nngg.corp\:11041
DEFAULT_ROWPREFETCH_SIZE=100
MONITOR_SQL=false
USE_SSL=true
USE_READ_ONLY_DS=true
JMS_NAMING_PROVIDER_URL=https-remoting\://BVMNGLCALYPD113.nngg.corp\:11101
CALYPSO_MESSAGING_SERVER_URL=failover\:(tcp\://localhost\:62616)?timeout\=30000
AUTO_FEED_EXTERNAL_REF=false
USE_SR_2022=True
LOG_CATEGORY=GestorSTPSwiftMessageEngine
SERVERHOST=BVMNGLCALYPD113.nngg.corp
EVENTSERVER_NAMING_PROVIDER_URL=https-remoting\://BVMNGLCALYPD113.nngg.corp\:11101
SKIP_INVENTORY_EXCEPTION=true
EDEALING_LLMEVENTSERVERURL=tcp\://localhost\:6199
DU_MESSAGING_SERVER_URL=tcp\://localhost\:61917
RATE_ENGINE_LLMEVENTSERVERURL=tcp\://localhost\:8199
CUSTOM_SSL_TRUSTSTORE_FILE=client.truststore